docs/make.jl

```diff
@@ -110,6 +110,25 @@ makedocs(modules=[DiffEqBase,DiffEqProblemLibrary,DiffEqBiological],
          ]
          ])
 
+#Redirect old links
+cd(@__DIR__) do
```

Comment:
The `.` in `walkdir` should refer to `build/`:

```suggestion
cd(joinpath(@__DIR__, "build")) do
```

Otherwise it might do stuff in `src/` etc. too.